docs/docs/add-custom-webpack-config.md

```diff
@@ -91,3 +91,45 @@ exports.onCreateWebpackConfig = ({ stage, actions }) => {
 ```
 
 You can always find more information on _resolve_ and other options in the official [Webpack docs](https://webpack.js.org/concepts/).
+
+### Modifying the babel loader
```

Comment:
I think this heading is a bit misleading - You are showing how to modify webpack rule here and not babel, right?

Reply:
@pieh thanks for checking out this PR!

This heading is the same as the one in the v1 docs that I linked: https://www.gatsbyjs.org/docs/add-custom-webpack-config/#modifying-the-babel-loader

I think it's reasonable because even though it's Webpack config, the goal is still to modify the `babel-loader` config (which Webpack will use).